src/sailing_robot/scripts/wind_direction_average

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+
+import math
+import collections
+import rospy
+from std_msgs.msg import Float32, Float64
+
+
+
+def angle_average(ang_list):
+    return math.atan2(sum([ math.sin(x*math.pi/180) for x in ang_list]),
+                      sum([ math.cos(x*math.pi/180) for x in ang_list]))*180/math.pi % 360
```

Comment:
There are functions `math.radians` and `math.degrees` to do the conversions - that may be more legible than doing the conversion by hand. I'd also like there to be a couple of tests for this function - the easiest way to do that is to put it into a module in `src/sailing_robot/src/sailing_robot` (...I know...) and follow the existing tests in `src/sailing_robot/tests`.